src/backends/kafka/mod.rs

```diff
@@ -423,24 +424,30 @@ mod tests {
         let topic = Topic {
             name: "test".to_string(),
         };
+
+        let my_callbacks: Box<dyn AssignmentCallbacks> = Box::new(EmptyCallbacks {});
+        consumer.subscribe(&[topic.clone()], my_callbacks).unwrap();
+
         let positions = HashMap::from([(
-            Partition {
-                topic: topic.clone(),
-                index: 0,
-            },
+            Partition { topic, index: 0 },
             Position {
                 offset: 100,
                 timestamp: Utc::now(),
             },
         )]);
 
-        let my_callbacks: Box<dyn AssignmentCallbacks> = Box::new(EmptyCallbacks {});
-        consumer.subscribe(&[topic], my_callbacks).unwrap();
```

Comment:
Can you use the callbacks to be signaled when assignment has happened?

Reply:
It does actually use that under the hood via the `consumer.tell()` call which it checks periodically